docker/Dockerfile

```diff
@@ -5,22 +5,25 @@
 #
 # To run:
 #	docker run -it --name niceman \
-#		-v /path/to/your/local/niceman:/niceman \
+#		-v $PWD:/root/niceman \
 #		-v /var/run/docker.sock:/var/run/docker.sock \
 #		niceman:latest
  
 FROM debian:jessie
 #FROM ubuntu:xenial
 
-COPY . /niceman
-COPY docker/niceman.cfg /root
-
 RUN apt-get update \
     && apt-get install -y build-essential libssl-dev libffi-dev vim wget \
     && apt-get install -y python-dev python-pyparsing python-crypto python-pip \
-    && pip install --upgrade pip \
-    && pip install setuptools==33.1.1 pytest docker-py boto3 dockerpty nose \
-    && pip install -e /niceman
+    && pip install --upgrade pip
+
+
+COPY . /root/niceman
+COPY docker/niceman.cfg /root
+
+# TODO: create a matching user here so we could bind mount and develop
+# within docker environment
```

Comment:
probably would require something like Dockerfile.in and a script, similar to what we did in simple_workflow